examples/starlark/validation/.expected/results.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 exitCode: 1
 items:
     - image: gcr.io/kpt-fn/starlark:unstable
-      stderr: 'fail: it is prohibited to have private key in a configmap'
+      stderr: '[error] /// : fail: it is prohibited to have private key in a configmap'
```

Comment:
was there a tracking issue for this?

Reply:
Created https://github.com/kubernetes-sigs/kustomize/pull/3955 to fix it upstream.